@here/harp-utils/lib/UrlUtils.ts

```diff
@@ -130,7 +130,10 @@ export function getUrlHostAndProtocol(
 
     const match = url.match(urlOriginRe);
     if (!match) {
-        throw new Error(`getUrlHostAndProtocol: unable to parse URL '${url}'`);
+        return {
```

Comment:
I don’t like silent failures, because it makes debugging much harder, because program execution continues in some invalid state and trying to find back to the point the program enters this state is hard.

I would prefer that `getUrlHostAndProtocol` stays the way it is and you just check the `window.location.href` (which anyway is the root cause of the issue) for an invalid value.